### ISSUE TITLE
Could you please push the js making slides to the github? 

### ISSUE THREAD

**Issue by LufengLIN** [MUSA611-CPLN692-spring2019/cpln692-week7#1]
Could you please push the js script making slides to the GitHub? It's hard to follow everything on class. Thanks.

**Comment by rossbernet** [comment]
Nathan posted this on Gitter at the end of class.

https://gitter.im/CPLN692-MUSA611-spring2019/community

But I just added it to this repo:

https://github.com/MUSA611-CPLN692-spring2019/cpln692-week7/tree/master/slides-example

@LufengLIN 
@LiZhuang214
